File: CPP/Abr_07_Strings_CPP/Concatenacao.cpp
#include <iostream>

using namespace std;

int main() {
  int i;
  char j;
  string maius = "", minus = "", nros = "";

  for(i=0; i<26; i++) {
    maius = maius + (char)('A' + i) + ' ';
    minus = minus + (char)('a' + i) + ' ';
  }

  for(j='0'; j<='9'; j++) {
    nros = nros + j;
    if (j != '9') {
       nros = nros + ", ";
    }
  }

  cout << minus << endl;
  cout << maius << endl << endl;
  cout << nros << endl;

  cout << endl << 'A' << (char)('A'+32) << endl;

  return(0);
}

File: POO_2026_1/src/Abr_07_Strings/Concatenacao.java
package Abr_07_Strings;

public class Concatenacao {

    public static void main(String[] args) {
      String minus = "", maius = "", nros = "";
      
      int i;
      
      for(i=0; i<26; i++) {
        minus = minus + (char)('a' + i);
        maius = maius + (char)('A' + i);
      }
      
      char j;
      for(j='0'; j<='9'; j++) {
        nros = nros + j;  
      }
      
      System.out.println(minus);
      System.out.println(maius);
      System.out.println(nros);      
    }

}
